internal/featuresupport/features_misc.go

```diff
@@ -287,16 +287,16 @@ func (f *skipMcoReboot) getIncompatibleFeatures(openshiftVersion string) *[]mode
 }
 
 func (f *skipMcoReboot) getIncompatibleArchitectures(openshiftVersion *string) *[]models.ArchitectureSupportLevelID {
-	return &[]models.ArchitectureSupportLevelID{
-		models.ArchitectureSupportLevelIDS390XARCHITECTURE,
-	}
+	return nil //&[]models.ArchitectureSupportLevelID{
+	//models.ArchitectureSupportLevelIDS390XARCHITECTURE,
+	//}
```

Comment:
Why returning nil? Also remove the comments

Reply:
Good catch ... forget to remove these lines after testing. Thank you.